resources/js/components/rich_text_editor.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import MUIRichTextEditor from 'mui-rte';
+import {stateToHTML} from 'draft-js-export-html';
+import debounce from 'lodash.debounce';
+
+const RichTextEditor = ({rteRef, rteProps, value, onChange}) => {
+    let handleChangeDebounce;
+
+    const handleChange = (state) => {
+        if (handleChangeDebounce) handleChangeDebounce.cancel()
+        handleChangeDebounce = debounce(() => {
+            const html = stateToHTML(state.getCurrentContent());
```

Comment:
bio text is not HTML , is MD

Reply:
lets take a look to https://www.npmjs.com/package/react-simplemde-editor

Reply:
done